golang/internal/sample.go

```diff
@@ -1,5 +1,36 @@
 package internal
 
+import (
+	"math"
+)
+
 func Summ(a, b int) int {
 	return a + b
 }
+
+func Cycle(xn, xk, xd float64) []float64 {
+	var numbers_cycle = []float64{}
+	for i := xn; i <= xk; i = i + xd {
+		numbers_cycle = append(numbers_cycle, Func_resolution(i))
+	}
+	return numbers_cycle
+}
+
+func Massive(numbers [5]float64) []float64 {
+	var numbers_massive = []float64{}
```

Comment:
```suggestion
	var numbers_massive = make([]float64, 0, len(numbers)) 
```
тогда не будет происходить несколько раз выделения нового массива под срез